ResearchKit/Common/ORKImageCaptureView.m

```diff
@@ -140,10 +157,18 @@ - (UIImage *)capturedImage {
 - (void)setError:(NSError * __nullable)error {
     _error = error;
     _headerView.alpha = error==nil ? 0 : 1;
-    _headerView.instructionLabel.text = [error.userInfo valueForKey:NSLocalizedDescriptionKey];
-    if (error && self.continueTitle) {
-        _continueSkipContainer.continueButtonItem.title = self.continueTitle;
-        _continueSkipContainer.skipButtonItem = nil;
+    _headerView.instructionLabel.text = error==nil ? nil : [error.userInfo valueForKey:NSLocalizedDescriptionKey];
+    
+    if (error) {
+        // Show skip, if available, and hide the template and continue/capture button
+        _previewView.templateImageHidden = YES;
+        _continueSkipContainer.continueButtonItem = nil;
+        _continueSkipContainer.skipButtonItem = _skipButtonItem;
+    } else {
+        // Reset to running state
+        _previewView.templateImageHidden = NO;
```

Comment:
Yep. For safety I imagine you could also call it from the other setters, for example the continue and skip item setters.